apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -999,15 +999,23 @@ impl Webserver {
                         with_spinner("Stopping containers", || {
                              let _ = docker::stop_containers(&project);
                         }, true);
+                    } else {
+                        log::info!("Received sigint, waiting for in-flight requests to complete");
+                        tokio::time::sleep(Duration::from_secs(30)).await;
```

Comment:
Should we make this 20s instead so that it doesn't some up on the deadline for k8s?

Reply:
Also would this affect the stop sown sequence in dev mode?

The guy in the talk had a sleep somewhere else?

Reply:
i wonder if it's possible to not hard code this value